v63003/u_inref/ydb297.csh

```diff
@@ -0,0 +1,35 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Tests that when over 31 locks hash to the same value, LOCK commands work correctly
+#
+$gtm_tst/com/dbcreate.csh mumps 1 >>& dbcreate.out
+# Simulates 2 errors in prior versions, either inappropriately seizes ownership of a lock or test hangs
+# Randomly chooses an option (1 will cause the test to hang, 0 will cause locks to be inappropriately seized)
+setenv hang `$gtm_tst/com/genrandnumbers.csh 1 0 1`
+$ydb_dist/mumps -run ydb297^ydb297 >>& job.out
+$gtm_tst/com/dbcheck.csh >>& dbcheck.out
+echo "# LOCKING RESOURCES THAT HASH TO THE SAME VALUE, EXPECT NUMBER OF LOCKS TO REMAIN CONSTANT ACROSS EACH RUN"
+echo "# INFO FROM RUN 1"
```

Comment:
Mention here that "we don't expect to see any FAILURE string here". And do just the grep. No need to pipe the output to cat and get the result into $status etc. Likewise with RUN2 and 3.